test.sh.in

```diff
@@ -223,10 +223,14 @@ done
 # Openscop inputs
 echo -e "\nTest with Openscop as input"
 echo "============================="
-TESTS_OPENSCOP="./test/gemver.scop"
+TESTS_OPENSCOP="\
+  @top_srcdir@/test/gemver.scop \
+  @top_srcdir@/test/simple-case-verification.scop \
+  @top_srcdir@/test/simple-case-computation.scop \
```

Comment:
These scop files need to have at least one CHECK line comment of the expected output.